<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_sale</class>
 <widget class="QWidget" name="add_sale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>405</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id договора</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_buyer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_flat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_id_buyer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id покупателя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_id_flat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id квартиры</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
